tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -3536,25 +3550,33 @@ def test_complex128(a):
 
     import numpy as np
 
-    integer = randint(1e6)
-    bl = np.bool(randint(1e5))
-    integer8 = np.int8(randint(0, 127))
-    integer16 = np.int16(randint(1e6))
-    integer32 = np.int32(randint(1e6))
-    integer64 = np.int64(randint(1e6))
-    fl = np.float(randint(1e6))
-    fl32 = np.float32(randint(1e6))
-    fl64 = np.float64(randint(1e6))
+    integer8 = randint(np.iinfo(np.int8(1)).min, np.iinfo(np.int8(1)).max, dtype=np.int8)
+    integer16 = randint(np.iinfo(np.int16(1)).min, np.iinfo(np.int16(1)).max, dtype=np.int16)
```

Comment:
instead of `np.iinfo(np.int8(1)).min` simply use `iinfo('int8').min`.
the same for the others, I don't see any meaning for `int8(1)` and i`nt16(1)`.